### DISCUSSION TITLE
Terms & Conditions concerning use of the DuckDB logo and presentation of the name DuckDB in third party products?

### DISCUSSION THREAD

**Discussion by rpbouman** [duckdb/duckdb#6202] [General]
Suppose I want to create a duckdb connector or component that exposes DuckDB functionality (for a plugin in a third party product). 

What are the terms and conditions concerning the use of the logo and presentation of the name DuckDB in such a connector/component?

For example, suppose I'm writing a plugin for an ETL tool that uses the DuckDB appender interface to load data in a table, would it be allowed to call that plugin a "DuckDB Appender"? 

Would such a component be allowed to use the DuckDb logo as its icon, or would one be allowed to use the DuckDb logo as part of a new Icon?

**Comment by hannes** [comment]
Its fine but please add a note like '"DuckDB" and the DuckDB Logo are registered trademarks of the DuckDB Foundation'.

**Reply by rpbouman** [reply]
All clear! Thanks for you reply.
